core/src/main/java/org/infinispan/factories/AbstractComponentRegistry.java

```diff
@@ -56,21 +57,24 @@
  * @deprecated Since 9.4, please use {@link BasicComponentRegistry} instead.
  */
 @SurvivesRestarts
-@Scope(Scopes.NAMED_CACHE)
+@Scope(Scopes.NONE)
```

Comment:
In this case it's more like ANY, because each subclass is in a different scope, but we still want the `@Scope` annotation in order to process the method annotations. Although in this particular case I think it was just to keep `@SurvivesRestarts`, I could delete both.